### PR TITLE
[plugin.audio.intergalacticfm] 3.0.0

### PR DESCRIPTION
**Pull request by PanderMusubi**
### Description
Existing plugin that was migration to Python 3.0 and some refactoring and smaller image sizes.

### Checklist:
- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

Note: could not get the fixes in this PR https://github.com/xbmc/repo-plugins/pull/3918 so had to create this new PR.

### CODE REVIEW

**Comment by basrieter** [comment]
Please make the add-on checker happy.

**Comment by PanderMusubi** [comment]
@basrieter done :-)

**Comment by PanderMusubi** [comment]
@basrieter can you move this one forward? Thanks

File: plugin.audio.intergalacticfm/addon.xml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+<requires>
+    <import addon="xbmc.python" version="3.0.0"/>
+    <import addon="script.module.requests" version="2.19.1"/>
+</requires>
+<extension point="xbmc.python.pluginsource" library="main.py">
+    <provides>audio</provides>
+</extension>
+<extension point="xbmc.addon.metadata">
+    <platform>all</platform>
+    <summary lang="en">No Station Such Dedication</summary>
```
Comment (enen92) [inline plugin.audio.intergalacticfm/addon.xml:12]: en_GB

File: plugin.audio.intergalacticfm/addon.xml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    <summary lang="de">No Station Such Dedication</summary>
```
Comment (enen92) [inline plugin.audio.intergalacticfm/addon.xml:13]: de_DE

File: plugin.audio.intergalacticfm/addon.xml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    <summary lang="nl">No Station Such Dedication</summary>
```
Comment (enen92) [inline plugin.audio.intergalacticfm/addon.xml:14]: nl_NL

File: plugin.audio.intergalacticfm/addon.xml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    <description lang="en"><![CDATA[
```
Comment (enen92) [inline plugin.audio.intergalacticfm/addon.xml:15]: en_GB (and remove CDATA)

File: plugin.audio.intergalacticfm/addon.xml [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+Audio add-on of listener-supported, commercial-free audio streams from Intergalacic FM.
+
+See also the video plugin. Please consider donating at https://intergalactic.fm to keep Intergalactic FM in orbit.
+    ]]></description>
+    <description lang="de"><![CDATA[
```
Comment (enen92) [inline plugin.audio.intergalacticfm/addon.xml:20]: de_DE

File: plugin.audio.intergalacticfm/addon.xml [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+Audioadd-on vom hörergetragenen, werbefreien Audioostreams von Intergalactic FM.
+
+Siehe auch die Video-Plugin. Bitte denk über eine Spende auf https://intergalactic.fm nach, um Intergalactic FM in der Umlaufbahn zu erhalte.
+    ]]></description>
+    <description lang="nl"><![CDATA[
```
Comment (enen92) [inline plugin.audio.intergalacticfm/addon.xml:25]: nl_NL

**Review by enen92 — CHANGES_REQUESTED** [review]
Just some minors on addon.xml

File: plugin.audio.intergalacticfm/addon.xml (hunk 1 above)
Comment (PanderMusubi) [reply plugin.audio.intergalacticfm/addon.xml:12]: Why `en_GB` and not `en_US`?

File: plugin.audio.intergalacticfm/addon.xml (hunk 2 above)
Comment (PanderMusubi) [reply plugin.audio.intergalacticfm/addon.xml:13]: This one will be removed.

File: plugin.audio.intergalacticfm/addon.xml (hunk 3 above)
Comment (PanderMusubi) [reply plugin.audio.intergalacticfm/addon.xml:14]: This one will be removed.

File: plugin.audio.intergalacticfm/addon.xml (hunk 5 above)
Comment (PanderMusubi) [reply plugin.audio.intergalacticfm/addon.xml:20]: This one will be removed.

File: plugin.audio.intergalacticfm/addon.xml (hunk 4 above)
Comment (PanderMusubi) [reply plugin.audio.intergalacticfm/addon.xml:15]: done

File: plugin.audio.intergalacticfm/addon.xml (hunk 1 above)
Comment (PanderMusubi) [reply plugin.audio.intergalacticfm/addon.xml:12]: done

Comment (enen92) [reply plugin.audio.intergalacticfm/addon.xml:12]: `en_GB` is kodi default. You haven't changed this...at least it doesn't show changed on the current revision.

**Review by enen92 — CHANGES_REQUESTED** [review]
See my previous comments, it doesn't look like you addressed any of them

**Comment by PanderMusubi** [comment]
While creating the PR, it was not picked up, will try to make a new one soon, thanks

**Comment by enen92** [comment]
Closing as stalled, please resubmit if still interested

**Pull request closed without merging**